src/Areas/Deploy/Commands/AzdAppLogGetCommand.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.Areas.Deploy.Options;
+using AzureMcp.Areas.Deploy.Services;
+using AzureMcp.Commands.Subscription;
+using AzureMcp.Services.Telemetry;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Areas.Deploy.Commands;
+
+public sealed class AzdAppLogGetCommand(ILogger<AzdAppLogGetCommand> logger) : SubscriptionCommand<AzdAppLogOptions>()
+{
+    private const string CommandTitle = "Get AZD deployed App Logs";
+    private readonly ILogger<AzdAppLogGetCommand> _logger = logger;
+
+    private readonly Option<string> _workspaceFolderOption = DeployOptionDefinitions.AzdAppLogOptions.WorkspaceFolder;
+    private readonly Option<string> _azdEnvNameOption = DeployOptionDefinitions.AzdAppLogOptions.AzdEnvName;
+    private readonly Option<int> _limitOption = DeployOptionDefinitions.AzdAppLogOptions.Limit;
+
+    public override string Name => "azd-app-log-get";
+
+    public override string Description =>
+        """
+        This tool helps fetch logs from log analytics workspace for Container Apps, App Services, function apps that were deployed through azd. Invoke this tool directly after a successful `azd up` or when user prompts to check the app's status or provide errors in the deployed apps.
```

Comment:
This tool fetches logs from the Log Analytics workspace for Container Apps, App Services, and Function Apps deployed using azd. Use it after a successful azd up to check app status or troubleshoot errors in deployed applications.
```suggestion
        This tool helps fetch logs from log analytics workspace for Container Apps, App Services, function apps that were deployed through azd. Invoke this tool directly after a successful `azd up` or when user prompts to check the app's status or provide errors in the deployed apps.
```